bundles/org.openhab.core.addon.marketplace.karaf/src/main/java/org/openhab/core/addon/marketplace/karaf/internal/community/CommunityKarafAddonHandler.java

```diff
@@ -148,7 +155,7 @@ private void installFromCache(String addonId) throws MarketplaceHandlerException
                             "The local cache folder doesn't contain a single file: " + addonPath, null);
                 }
                 try {
-                    karService.install(bundleFiles.get(0).toUri(), true);
+                    karService.install(bundleFiles.get(0).toUri(), false);
```

Comment:
From my understanding `true` installs the contained features. That’s what I would expect after installing an addon from the marketplace. Am I wrong here?

Reply:
It is actually the otherway around which is confusing :-) 

```java
    /**
     * Install KAR from a given URI
     *
     * @param karUri URI of the kar to install
     * @param noAutoStartBundles true to not start automatically the bundles, false else
     * @throws Exception in case of installation failure
     */
    void install(URI karUri, boolean noAutoStartBundles) throws Exception;
```